electron/shortcuts/nonesteamgame/nonesteamgame.ts

```diff
@@ -342,10 +326,11 @@ async function addNonSteamGame(props: {
 
     const message = i18next.t('notify.finished.add.steam.corrupt', {
       defaultValue:
-        '{{game}} could not be added to all found Steam users. See logs for more info.  A restart of Steam is required, changes to take effect.',
+        '{{game}} could not be added to all found Steam users. See logs for more info. A restart of Steam is required, changes to take effect.',
```

Comment:
I noticed this string `A restart of Steam is required, changes to take effect` when adding translations for spanish the other day. I think `A restart of Steam is required for changes to take effect` makes more sense, it sounds like there's a missing word for me there (this is in many of the strings).